deploy/cr.yaml

```diff
@@ -48,6 +48,7 @@ spec:
 #      runAsGroup: 1001
 #      supplementalGroups: [1001]
 #    serviceAccountName: percona-xtradb-cluster-operator-workload
+#    imagePullPolicy: Always
```

Comment:
can we add changing of this field to some exiting test?
e.g. `limits` test

Reply:
added https://github.com/percona/percona-xtradb-cluster-operator/pull/545/commits/258fd93e2579a53930e2ef947768a8fbd9b823e0